src/Data/List/Relation/Binary/Sublist/Heterogeneous/Properties.agda

```diff
@@ -400,26 +400,22 @@ module Antisymmetry
 
   open ℕ.≤-Reasoning
 
+  private
+    antisym-lemma : ∀ xs ys₁ y → Sublist R xs ys₁ → ¬ Sublist S (y ∷ ys₁) xs
+    antisym-lemma xs ys₁ y rs ss = ℕ.<-irrefl ≡.refl (begin
+      length (y ∷ ys₁) ≤⟨ length-mono-≤ ss ⟩
+      length xs        ≤⟨ length-mono-≤ rs ⟩
+      length ys₁       ∎)
```

Comment:
Regardless of possible reuse (could be treated downstream?), suggest to use the `variable`s already declared, and `_$_` to reduce the parenthesis cognitive load:
```suggestion
    antisym-lemma : Sublist R xs ys → ¬ Sublist S (y ∷ ys) xs
    antisym-lemma {xs} {ys} {y} rs ss = ℕ.<-irrefl ≡.refl $ begin
      length (y ∷ ys) ≤⟨ length-mono-≤ ss ⟩
      length xs       ≤⟨ length-mono-≤ rs ⟩
      length ys       ∎
```